### DISCUSSION TITLE
not found: patch

### DISCUSSION THREAD

**Discussion by webmapLee** [pnpm/pnpm#5275] [Q&A]
when i want to patcha a package, i exeute 
`
pnpm patch is-odd@3.0.1
`
show the follow erro:
![image](https://user-images.githubusercontent.com/20334500/187082113-726ee341-0683-4cab-bdbb-6500de531492.png)

i don't know why

env:
OS: windows11
pnpm:6.32.3
node:16.14.0
npm:8.3.1

**Comment by zkochan** [comment]
The patch command was only added in pnpm v7.4

**Comment by webmapLee** [comment]
yes,  but  i  can't  upgrade my pnpm,  i install pnpm  by npm , and  updgrade by npm
![image](https://user-images.githubusercontent.com/20334500/187213780-5e8b5279-e2a5-4a10-ab86-ca30e76eed33.png)

**Reply by zkochan** [reply]
Try to remove pnpm manually https://pnpm.io/installation#troubleshooting

**Comment by webmapLee** [comment]
ok , thx
